dih_models/parameters.py

```
@@ -14433,9 +14433,389 @@ def calculate_personal_lifetime_wealth_conservative_baseline(
+    compute=lambda ctx: ctx["STATE_RTT_PROVEN_TREATMENTS_ANNUAL"] * ctx["STATE_RTT_EVALUATIONS_QUEUE_SHARE"] * (ctx["RARE_DISEASE_PATIENTS_GLOBAL"] / ctx["RARE_DISEASES_COUNT_GLOBAL"]) * ctx["STATE_RTT_EFFECTIVE_TREATMENT_DALY_GAIN"],
+    latex_symbol=r"DALY_{delay}",
+)
+
+STATE_RTT_INACTION_VALUE_PER_DELAY_YEAR = Parameter(
+    float(STATE_RTT_INACTION_DALYS_PER_DELAY_YEAR) * float(STANDARD_ECONOMIC_QALY_VALUE_USD),
+    manual_ref="knowledge/appendix/state-right-to-trial-impact.qmd",
+    source_type="calculated",
+    description="Economic value of the permanent annual global benefit forfeited per year the 50-state system is delayed, at the standard economic value of a healthy life-year.",
+    display_name="Economic Value Forfeited per Year of Delay",
+    unit="USD/year",
+    formula="STATE_RTT_INACTION_DALYS_PER_DELAY_YEAR × STANDARD_ECONOMIC_QALY_VALUE_USD",
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)